changelogs/fragments/327-pause-on-restore.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - Pause hosted-engine deployment on failure of engine-backup --mode=restore (https://github.com/oVirt/ovirt-ansible-collection/pull/327).
```

Comment:
This should start with the role name (for example https://github.com/oVirt/ovirt-ansible-collection/tree/master/changelogs#example)

Reply:
Done